lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -169,7 +169,7 @@ def test_pathclip():
 # test mixed mode rendering
 @needs_pgf_xelatex
 @pytest.mark.backend('pgf')
-@image_comparison(['pgf_mixedmode.pdf'], style='default')
+@image_comparison(['pgf_mixedmode.pdf'], style='default', tol=0.9)
```

Comment:
Is there a reason this tolerance bump is not conditional like the rest of them?

Reply:
Because it probably shouldn't have been there. As noted above, this test doesn't appear in the list. I think this may have been for different Ghostscript versions, but is unnecessary now.